utils/checkRoles.js

```diff
@@ -1,18 +1,1266 @@
-const { readFileSync } = require("fs");
 const { Client, GatewayIntentBits, EmbedBuilder } = require("discord.js");
+const {
+  registry,
+  ChainId,
+  Transmuter__factory,
+  ANGLE__factory,
+  ProposalSender__factory,
+  GaugeController__factory,
+  SmartWalletWhitelist__factory,
+  VeANGLE__factory,
+  VeBoostProxy__factory,
+  MerklGaugeMiddleman__factory,
+  Distributor__factory,
+  DistributionCreator__factory,
+  Savings__factory,
+  VaultManager__factory,
+  Treasury__factory,
+  ProxyAdmin__factory,
+  ProposalReceiver__factory,
+  AngleRouterV2__factory,
+} = require("@angleprotocol/sdk");
+const { ethers } = require("ethers");
+const { createPublicClient, http, getContract } = require("viem");
+
+let actors = {};
+let roles = [];
+let listAddressToCheck = [];
+
+const LayerZeroBridgeTokenAbi = [
```

Comment:
we have those in the sdk no?

Reply:
We don't unfortunately or not all functions I need

Reply:
Worth adding them then, as it can be useful for multiple repos